tests/wpt/metadata/html/semantics/scripting-1/the-script-element/fetch-src/alpha/base.html.ini

```diff
@@ -1,5 +1,5 @@
 [base.html]
   type: testharness
   [Script src with a base URL]
-    expected: FAIL
+    expected: PASS
```

Comment:
No need to explicitly mark it as PASS; if the test passes, you can just remove the `expected: ` key-value pair. Since there's only one expected failure in this file previously, the entire file can be removed as well.